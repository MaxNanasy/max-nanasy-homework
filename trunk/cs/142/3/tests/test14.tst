   void sym1 () {
   }
   void sym1 () {
   }
