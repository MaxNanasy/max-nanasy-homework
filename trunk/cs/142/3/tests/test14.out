Error: SEMANTIC: REDEFINED SYMBOL
***** Symbol Table Contents *****
N: int C: TYPE T: null
N: void C: TYPE T: null
N: short C: TYPE T: null
N: sym1 C: PROCEDURE T: void
